.ci/bump_license_year.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+CURR_YEAR=$(date +"%Y")
+JAVA_FILE=$(find ./src/main -type f -name *.java -print -quit)
+echo "Current year will be taken from $JAVA_FILE"
+PREV_YEAR=$(grep "Copyright" $JAVA_FILE | cut -d " " -f 4 | cut -d '-' -f 2)
+OLD_PATTERN="// Copyright (C) 2001-$PREV_YEAR the original author or authors."
+NEW_PATTERN="// Copyright (C) 2001-$CURR_YEAR the original author or authors."
+find ./src -type f \( -name *.java -o -name *.header -o -name *.g \) \
+  -exec sed -i "s|$OLD_PATTERN|$NEW_PATTERN|g" {} +
```

Comment:
Since this is a regular expression pattern, don't you have to sanitize the `()`s and `-`?

Reply:
changed name of variable, it is not a regexp pattern

Reply:
The `s||g` looks like a regular expression text to me.

Reply:
`sed -E` is for regexp. we use just search and replace.